### ISSUE TITLE
BUG autocomplete not working after minus sign

### ISSUE THREAD

**Issue by jkokorian** [ipython/ipython#5836]
Code completion after a '.' does not work if the the thing that precedes the dot is itself preceded by a minus sign.

Example:

``` python
class Test(object):
    def __init__(self):
        self.number = 10.0

t = Test()
t. #pressing tab correctly inserts 'number'

-t. #pressing tab doesn't do anything

- t. #with a space between the variable and the minus sign pressing tab works fine
```

**Comment by Carreau** [comment]
This will be fixed by #10182.

**Comment by jkokorian** [comment]
awesome!

**Comment by Carreau** [comment]
Closed by #10182
